### DISCUSSION TITLE
Still being supported?

### DISCUSSION THREAD

**Discussion by theresaway** [deadcast2/UltraEd#165] [Q&A]
Is this still being supported? I'd love to make some N64 games.

**Comment by deadcast2** [comment]
Hi @ACIDULATED!

I haven't given up working on this project but I'm pretty busy now working on a commercial game and developing a game engine for modern computers. Maybe as I learn things from my other projects I'll incorporate those into UltraEd or when I get some more time and energy. I think most things that need to be done are in the Issues list. It's not an exhaustive list but if all completed it would make it pretty useful. Over time I'll probably hop back onto UltraEd but not sure when yet. I've been working on this project for so long that I wish somebody else would progress it for a while lol! :D j/k
